piker/watchlists.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+from collections import defaultdict
+
+from .log import get_logger
+
+log = get_logger(__name__)
+
+def write_sorted_json(watchlist, path):
+    for key in watchlist:
+        s = set(watchlist[key])
+        watchlist[key] = list(s)
+        watchlist[key].sort()
+    with open(path, 'w') as f:
+        json.dump(watchlist, f, sort_keys=True)
+
+
+def make_config_dir(dir_path):
+    if not os.path.isdir(dir_path):
+        log.debug(f"Creating config dir {dir_path}")
+        os.makedirs(dir_path)
+
+
+def ensure_watchlists(file_path):
+    mode = 'r' if os.path.isfile(file_path) else 'w'
+    with open(file_path, mode) as f:
+        data = json.load(f) if not os.stat(file_path).st_size == 0 else {}
+    return data
+
+
+def load_watchlists(watchlist, path):
```

Comment:
Shouldn't this be called `write_watchlists()`?
Confused how it's *loading* anything.

Reply:
i was referring to loading the watchlist for the command line

Reply:
Yeah from the CLI it's one thing to describe how the user *should use it* versus what calling this function *does*.